### PR TITLE
update various libraries

### PR DESCRIPTION
**Pull request by chengjie8**
## How does this fix it?[^1]
upgrade library versions 

## How to test this PR
- Run github scan
- Run e2e test

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: api-gateway/build.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-webflux'
   implementation 'org.springframework.boot:spring-boot-starter-actuator'
   implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+  implementation 'org.bouncycastle:bcpkix-jdk18on:1.74'
```
Comment (yoomlam) [inline api-gateway/build.gradle:12]: Indirect dependencies should be declared as constraints. See https://github.com/department-of-veterans-affairs/abd-vro/blob/1ab3968ce669af1de76d8c7238de8d4e1b1cfed2/gradle-plugins/src/main/groovy/shared.java.vro-dep-constraints.gradle#L27

File: svc-bie-kafka/build.gradle [hunk 2]
```diff
@@ -32,7 +32,7 @@ dependencies {
 
   // RabbitMQ
   implementation 'org.springframework.boot:spring-boot-starter-amqp'
-  implementation 'org.springframework.kafka:spring-kafka'
+  implementation 'org.springframework.kafka:spring-kafka:2.9.11'
```
Comment (yoomlam) [inline svc-bie-kafka/build.gradle:35]: Set this version in https://github.com/department-of-veterans-affairs/abd-vro/blob/1ab3968ce669af1de76d8c7238de8d4e1b1cfed2/gradle-plugins/src/main/groovy/shared.java.vro-dep-constraints.gradle#L27

**Comment by chengjie8** [comment]
I have addressed @yoomlam's review comments.

**Review by tejans24 — APPROVED** [review]
LGTM

**Review by yoomlam — APPROVED** [review]

**Pull request merged**